app.js

```diff
@@ -72,10 +73,30 @@ if (process.env.NODE_ENV !== 'production') {
   require('./webpack/webpack')(app);
 }
 
+// get the base js filename
+const publicJsPath = `${__dirname}/public/js`;
+let baseJsFile;
+if (process.env.NODE_ENV === 'production') {
+  const files = fs.readdirSync(publicJsPath);
+  console.log(files);
```

Comment:
please remove this debug information

Reply:
this has already remove on https://github.com/steemit/faucet/pull/481/commits/12a985d822f999b1151f9d13246d10038efd5cf8